project1/CLI_strings/main.c

```diff
@@ -0,0 +1,35 @@
+#include "main.h"
+
+
+int main(int argc, char *argv[]) {
+
+	///*handling main arguments*/
+	if (argc != 3) {
+		printf("Wrong number of arguments, expected 2\n");
+		return EXIT_FAILURE;
+	}
+	FILE* file;
+	/*variables declaraion*/
+	char read_buff[BUFF_SIZE];
+
+	/*getting file to program's memory*/
+	if ((file = fopen(argv[1], "r")) == NULL) {
+		printf("File opening error!\n");
+		return EXIT_FAILURE;
+	}
+	fgets(read_buff, BUFF_SIZE,file);
+	fclose(file);
+	/*spliting strings to separate lines*/
+	char* token;
```

Comment:
**nit**: name **token** is confusing. `strtok()` returns a pointer to a string section not a single character or token. Consider renaming it to something like **current_section**.